### PR TITLE
Release: add Gitlab pipeline for pushing helm charts to NGC

### PR DESCRIPTION
**Pull request by ArangoGutierrez**

### CODE REVIEW

**Comment by ArangoGutierrez** [comment]
Needs: #121

File: .nvidia-ci.yml [hunk 1]
```diff
@@ -164,6 +171,26 @@ release:ngc:
...
+  before_script:
+    - !reference [.ngccli-setup, before_script]
+    - !reference [.helmcli-setup, before_script]
+    - apt-get update && apt-get install -y make
+  script:
+    - 'echo "Pushing the helm chart to NGC"'
+    - helm lint deployments/helm/*
+    - make helm-chart
+    - export VERSION=$(awk -F= '/^VERSION/ { print $2 }' versions.mk | tr -d '[:space:]')
+    - ngc-cli/ngc registry chart push --source k8s-nim-operator-${VERSION}.tgz --org nvidia --team no-team nvidia/k8s-nim-operator:${VERSION}
```
Comment (shivamerla) [inline .nvidia-ci.yml:192]: @ArangoGutierrez did you try this with staging to verify?

File: .common-ci.yml [hunk 2]
```diff
@@ -28,6 +28,7 @@ stages:
   - test
   - lint
   - helm-lint
+  - helm
```
Comment (shivamerla) [inline .common-ci.yml:31]: we should publish helm chart after the image has been successfully published. i.e this stage should be part of release.

Comment (ArangoGutierrez) [reply .common-ci.yml:31]: Done, fixed

**Comment by shivamerla** [comment]
@ArangoGutierrez  can you cleanup the chart (.tgz) from the PR.

**Comment by ArangoGutierrez** [comment]
> @ArangoGutierrez can you cleanup the chart (.tgz) from the PR.

ups done

**Review by shivamerla — APPROVED** [review]
lgtm

**Pull request merged**